tdesign/src/Layout/PaddedSingleCol/PaddedSingleCol.tsx

```diff
@@ -19,8 +19,6 @@ const PaddedSingleCol = ({
   children,
   className,
   innerClassName,
-  leftPadding,
-  rightPadding,
```

Comment:
If we are not using those props, we should remove them from the `IPaddedSingleColProps` too to avoid misleading consumers of this component :slightly_smiling_face:

Reply:
Good catch! both removed from the interface